src/mappings/talent-layer-escrow.ts

```diff
@@ -71,7 +72,6 @@ export function handleTransactionCreated(event: TransactionCreated): void {
   proposal.status = 'Validated'
   proposal.save()
 
-  service.status = 'Confirmed'
```

Comment:
Redundant, this is already handled in "getOrCreateService()" L68

Reply:
No it's not Confirmed is only after proposal validated

Reply:
True, got mixed up with the "Filled"  old status